java/src/main/java/org/rocksdb/WriteBatch.java

```diff
@@ -106,6 +106,12 @@ public Handler() {
     public abstract void delete(byte[] key);
     public abstract void logData(byte[] blob);
 
+    public void singleDelete(byte[] key) {}
+    public void singleDelete(int columnFamilyID, byte[] key) {}
```

Comment:
I struggled with this one too.  There are a few issues in my mind:
     * Consistency with the C++ API. 
     * Use cases. The main use case for me, is to parse the WAL data to replay in a different process/node. So the handle is not needed on the reader side. 
     * I didn't find any particular convenient API to map a cf_id to a cf_family handle. Ideally from a rocksdb handle (open a db, enumerate all the column families etc.). 

Please let me know what you think.